app/assets/stylesheets/components/_form.sass

```diff
@@ -0,0 +1,41 @@
+$Form-border-input: Theme-color(input, border)
+$Form-color-filled: Theme-color(input, filled)
+$Form-color-input-filled: Theme-color(input, filled)
+$Form-color-input-placeholder: Theme-color(input, placeholder)
+$Form-color-label: Theme-color(text, label)
+$Form-fontSize-input: Theme-fontSize(text, base, font-size)
+$Form-height: 40px
+$Form-input-border: 2px
+$Form-input-width-hashtag: 212px
+$Form-spacing-base: $Theme-spacing-base
+$Form-spacing-small: $Theme-spacing-small
+$Form-spacing-tiny: $Theme-spacing-small / 2
+$Form-width: 425px
+
+.Form
+  margin: 0 auto
+  width: $Form-width
+
+.Form-label
+  color: $Form-color-label
+  display: block
+  padding: $Form-spacing-base 0 $Form-spacing-tiny 0
+
+.Form-input
+  border-radius: $Form-input-border
+  border: 2px solid $Form-border-input
+  font-color: $Form-color-filled
```

Comment:
This property is not valid CSS.
I've told you many times: Don't make CSS previous to see how it renders in the browser and you feel the actual need to implement things in CSS.
This proves you do things without actually paying attention to what each individual property does.
You're learning, so you need to pay extra attention to these details.